scripts/travis-before-script.py

```diff
@@ -0,0 +1,17 @@
+import codecs
```

Comment:
It's not my favorite. On the plus side, it makes use of the Jinja templates we have for Ansible. But, it adds another discrepancy between Travis CI and dev environments. I was searching for a way to running ansible on Travis CI, but all I could find were guides to test ansible **roles** with Travis.

Though I recall we might be able to make an inventory file just for Travis with a single "localhost" entry? I'll try that too.